cimsparql/model.py

```diff
@@ -135,6 +135,8 @@ def bus_data(
         with_market: bool = True,
         with_dummy_buses: bool = False,
         container: bool = False,
+        delta_power: bool = True,
```

Comment:
**Similar**: can `delta_power` alwauys be there (possibly `optional` in query). `network_analysis` is probably OK to have as argument since it at the moment relies on the `SN` schema